lib/node_modules/@stdlib/math/base/tools/evalrational-compile-c/lib/main.js

```diff
@@ -51,7 +53,14 @@ var NAN_TEMPLATE = readFile( join( dir, 'nan.c.txt' ), opts );
 * @returns {string} serialized value
 */
 function value2string( x ) {
-	var str = x.toString();
+	var str;
+	if ( x === PINF ) {
+		return '1.0' + {{dtype_suffix}} + ' / 0.0' + {{dtype_suffix}};
+	}
+	if ( x === NINF ) {
+		return '-1.0' + {{dtype_suffix}} + ' / 0.0' + {{dtype_suffix}};
```

Comment:
Sorry. I wasn't thinking. The disadvantages of writing code via the web UI.

```suggestion
		return '1.0{{dtype_suffix}} / 0.0{{dtype_suffix}}';
	}
	if ( x === NINF ) {
		return '-1.0{{dtype_suffix}} / 0.0{{dtype_suffix}}';
```